packages/components/ng-ovh-otrs/rollup.config.js

```diff
@@ -0,0 +1,18 @@
+import configGenerator from '@ovh-ux/component-rollup-config';
+
+const config = configGenerator({
+  input: './src/index.js',
+});
+
+export default [
+  config.es(),
```

Comment:
Probably not yours, but sometimes there is a check on the targeted environment to know which kind of dist should be made, maybe we could uniformize

Reply:
Thanks for pointing out this.
The rollup configuration has been updated.